docs/.vuepress/config.js

```diff
@@ -33,6 +33,7 @@ module.exports = {
             '/extend/data',
             '/extend/distribution',
             '/extend/update-b8',
+            '/extend/update-b10',
```

Comment:
Should we remove the old upgrade guide for beta.8? At least from the menu?

Reply:
@franzliedke I'd leave it be, could be useful for old extensions that developers want to revive.